client/my-sites/upgrades/domain-management/controller.jsx

```diff
@@ -28,9 +29,9 @@ import titleActions from 'lib/screen-title/actions';
 const productsList = new ProductsList(),
 	sites = new SitesList();
 
-const renderPage = function( component ) {
+const renderPage = function( component, context ) {
 	ReactDom.render(
-		component,
+		React.createElement( Provider, { store: context.store }, component ),
```

Comment:
@rralian for context, we are having the same conversation right now at https://github.com/Automattic/wp-calypso/pull/2018#discussion_r48819960